scripts/install-freetype

```diff
@@ -0,0 +1,28 @@
+#CLEAN IF ANY
+rm -rf $(grep -Ril "freetype" /opt/devkitpro/portlibs/nds/)
+
+#INSTALL
+pkgver=$1
+
+wget http://download.savannah.gnu.org/releases/freetype/freetype-$pkgver.tar.gz
+tar -xvf freetype-$pkgver.tar.gz
+
+cd freetype-$pkgver
+source /opt/devkitpro/ndsvars.sh
+./configure --prefix="${PORTLIBS_PREFIX}" --host=arm-none-eabi \
+--disable-shared --enable-static \
+--with-zlib \
+--with-bzip2 \
+--with-png
+make
+
+
+make DESTDIR="$pkgdir" install
+
+sed 's/\$show_static/yes/' -i "${pkgdir}/${PORTLIBS_PREFIX}/bin/freetype-config"
+
+install -Dm644 "docs/LICENSE.TXT" "${pkgdir}/${PORTLIBS_PREFIX}/licenses/${pkgname}/LICENSE.TXT"
+install -Dm644 "docs/GPLv2.TXT"   "${pkgdir}/${PORTLIBS_PREFIX}/licenses/${pkgname}/GPLv2.TXT"
+install -Dm644 "docs/FTL.TXT"     "${pkgdir}/${PORTLIBS_PREFIX}/licenses/${pkgname}/FTL.TXT"
```

Comment:
No need for this, we are not officially releasing anything. "Install" means putting the files where we can locally build against them.